### PR TITLE
Add movie details route, view and controller

### PR DESCRIPTION
**Pull request by keyxyz**

### CODE REVIEW

File: resources/views/movie/front/moviedetails.blade.php [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+			<!-- end dropdown -->
+		</ul>
+		<!-- end menu nav -->
+	</div>
+	<!-- end mobile menu -->
+
+	<!-- details -->
+	<section class="section section--details">
+		<!-- details background -->
+		<div class="section__details-bg" data-bg="img/bg/details__bg.jpg"></div>
```
[the remark this answers is not in the document]
Comment (keyxyz) [reply resources/views/movie/front/moviedetails.blade.php]: Alright, Thanks

File: resources/views/movie/front/moviedetails.blade.php [hunk 2]
```diff
@@ -123,26 +123,26 @@
...
 						<!-- card content -->
 						<div class="item__content">
 							<div class="item__wrap">
-								<span class="item__rate">8.4</span>
+								<span class="item__rate">{{ $movie->rating }}</span>
 
 								<ul class="item__list">
-									<li>Full HD</li>
-									<li>16+</li>
+									<li>{{ $movie->video_quality }}</li>
```
Comment (MaablabsTech) [inline resources/views/movie/front/moviedetails.blade.php:136]: The video quality has a hasOne relationship that you must reference to.

File: resources/views/movie/front/moviedetails.blade.php [hunk 3]
```diff
@@ -123,26 +123,26 @@
...
+									<li>{{ $movie->pg_rate }}</li>
 								</ul>
 							</div>
 
 							<ul class="item__meta">
 								<li><span>Genre:</span> <a href="#">Action</a> <a href="#">Triler</a></li>
```
Comment (MaablabsTech) [inline resources/views/movie/front/moviedetails.blade.php:142]: Use the genres hasMany relationship here.

**Comment by keyxyz** [comment]
I could have known but I have not copy of the ER diagram. There's no way I could know. 

Thanks, I'll do.

File: app/Http/Controllers/MovieController.php [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+class MovieController extends Controller
+{
+    public  movie_details(string $slug) {
+        
+        $movie = Movie::where('slug', $slug)->get();
+
+        if (!is_null($movie)) {
+            return view('movie.detail')->with($movie);
+        } else {
+            return view('home.index');
```
Comment (MaablabsTech) [inline app/Http/Controllers/MovieController.php]: Please change the else to return a redirection to '/'

File: app/Http/Controllers/MovieController.php [hunk 5]
```diff
@@ -7,14 +7,14 @@
...
-    public  movie_details(string $slug) {
+    public movie_details(string $slug) {
         
         $movie = Movie::where('slug', $slug)->get();
 
         if (!is_null($movie)) {
             return view('movie.detail')->with($movie);
         } else {
-            return view('home.index');
+            redirect('/');
```
Comment (MaablabsTech) [inline app/Http/Controllers/MovieController.php:18]: return redirect("/");

Comment (keyxyz) [reply app/Http/Controllers/MovieController.php]: @MaablabsTech I just did it

**Review by MaablabsTech — APPROVED** [review]

**Pull request merged**